ADVANCED.md

```diff
@@ -529,6 +543,7 @@ page for suggestions, and maintaining the default behavior when selecting a resu
 | Name                          | type                                                                         | Required? | Default                                                            | Options | Description                                                                                                                                                                                                                                                                                                                                                  |
 | ----------------------------- | ---------------------------------------------------------------------------- | --------- | ------------------------------------------------------------------ | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | className                     | String                                                                       | no        |                                                                    |         |                                                                                                                                                                                                                                                                                                                                                              |
+| clearFilters                  | Boolean                                                                      | no        | true                                                               |         | Should existing filters be cleared when a new search is performed?                                                                                                                                                                                                                                                                                           |
```

Comment:
Sorry, I think this one confuses me a bit. Just to clarify: we're adding *two* new places to pass `clearFilters` - one within `setSearchTerm` options and one on the `<SearchBox>` component? So usage looks like this:

`setSearchTerm('foo', { clearFilters: true })`
`<SearchBox clearFilters />`

Is that correct?

EDIT: Ah okay sorry, I just read further down in the PR description and I see that described in the changelog now. One thing I might suggest is adding an example section somewhere (either in the ADVANCED.md file or a specific doc for SearchBox, perhaps) what that behavior looks like. Although it also might be better to save that for when we switch to a documentation generator like Styleguidist or Storybook :)

Reply:
Also, one thing I wanted to quickly note - `clearFilters()` is a function/action elsewhere but a boolean here. To my mind, adding multiple variables with the same name and having them be distinctly different types is pretty confusing. I would personally suggest `shouldClearFilters` instead to make it clear which is a bool and which is a function, but that's just my [preference for naming booleans](https://www.samanthaming.com/tidbits/34-better-boolean-variable-names). :)

Reply:
@constancecchen That is a *great* point. One that you have mentioned before but I clearly forgot. Thank you, I will make this change 👍